src/index.jsx

```diff
@@ -45,6 +45,7 @@ import isEnabled from './helpers/featureFlags';
 import sessionMonitor from './SessionMonitor';
 import Workspace from './Workspace';
 import ResourceBrowser from './ResourceBrowser';
+import DiscoveryBeta from './Discovery';
```

Comment:
Is there a strategic value to including "Beta" in the name of this component?

Reply:
Woops, no I forgot to update that.